pkgs/tools/text/fastmod/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fastmod";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "facebookincubator";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0089a17h0wgan3fs6x1la35lzjs1pib7p81wqkh3zcwvx8ffa8z8";
+  };
+
+  cargoSha256 = "0i5i9hsqynb7c1wv63p7y9k7yfl7iz67favrml97gy8l0dzi0666";
```

Thanks for making all the changes and fixing up the git history. It seems that the `cargoSha256` hash is still wrong, leading to the ofborg failure (I get the same error on my machine). With the following change it should be fine:

```suggestion
  cargoSha256 = "02nkxjwfiljndmi0pv98chfsw9vmjzgmp5r14mchpayp4943qk9m";
```

---

Sorry, I should've tried "nix-build" after making those changes. I've accepted your suggestion and squashed the changes again. PTAL.

PS: nix-build worked for me locally.